### DISCUSSION TITLE
Droid Slam Implementation in Gazebo

### DISCUSSION THREAD

**Discussion by hamzahsohail** [princeton-vl/DROID-SLAM#80] [Q&A]
I have successfully implemented DROID-SLAM on given datasets as well as my own dataset. 

Now I want to run this as a robot simulation in Gazebo. Help required.

**Comment by Aurora-chevalier** [comment]
Hello, I can't read the image when training the tartan dataset, can you show the instructions you used to train the tartan dataset？
![1](https://user-images.githubusercontent.com/112998806/223696742-7ebee081-2eff-4014-9a2b-f19b577359f6.jpg)

**Reply by hamzahsohail** [reply]
We did not train on any dataset, we ran on pre trained models/datasets.
